src/hooks/useSendAnalyticsData.ts

```diff
@@ -44,6 +44,7 @@ function useSendAnalyticsData(): (data: IAnalyticsData) => void {
             }
 
             send(event, trackData)
+            window.dataLayer = []
```

Comment:
```suggestion
```

DO NOT touch dataLayer - it should be managed by the Google Provider inside our AnalyticsSDK (`@gooddollar/web3sdk-v2`).

Describe the issue, we will discuss it

Reply:
The issue is described in the original issue https://github.com/GoodDollar/GoodProtocolUI/issues/268
where values are someway 'cached' and get send with new following events (I guess properties not defined in the other ones?)
The proposed solution to clear dataLayer came from @drptiago 

I will properly re-test this, but if you have other suggestion to tackle this..

Reply:
@L03TJ3 should be properly fixed at **monorepo** side

Reply:
I reverted this from here